pxteditor/events.ts

```diff
@@ -0,0 +1,46 @@
+namespace pxt.events {
+
+    export type EditorType = 'blocks' | 'js';
```

Comment:
to stay consistent, use file extension, ``.ts``

Reply:
Sounds good.